packages/flame/lib/src/game/camera.dart

```diff
@@ -155,8 +155,8 @@ class Camera extends Projector {
     if (_transform.m11 == zoom &&
         _transform.m22 == zoom &&
         _transform.m33 == zoom &&
-        _transform.m14 == translateX &&
-        _transform.m24 == translateY) {
+        _transform.m41 == translateX &&
+        _transform.m42 == translateY) {
```

Comment:
Hmm, how does this make sense actually. (But it works...)
Do you know @wolfenrain? With `mXY` where X represents row and Y represents column this would put the values of the translation on the last row, but the translate values should be on the last column right?
```
1  0  0  tx
0  1  0  ty
0  0  1  0 
0  0  0  1
```
Where `tx` and `ty` are the translate values.
But with the changed code I'm expecting it to to check these values:
```
1  0  0  0
0  1  0  0 
0  0  1  0
tx ty 0  1

Reply:
Nvm me, it is supposed to be like the bottom matrix.

Reply:
Glad I didnt had to break my brain over this haha